packages/react/src/components/Search/Search.js

```diff
@@ -162,9 +162,12 @@ export default class Search extends Component {
     const CloseIconX = size === 'xl' ? Close20 : Close16;
 
     return (
-      <div className={searchClasses}>
+      <div
+        role="search"
+        aria-labelledby={id + '-search'}
+        className={searchClasses}>
         <Search16 className={`${prefix}--search-magnifier`} />
-        <label htmlFor={id} className={`${prefix}--label`}>
+        <label id={id + '-search'} htmlFor={id} className={`${prefix}--label`}>
```

Comment:
Maybe it'd help out to create a variable like `const searchId = `${id}-search`;` and use it in both spots? Just so updates to one location are mirrored in the other location.

Reply:
Good idea! I'll add it!